make: *** [manifests] Error 1 while running quick_install.sh

/kind bug

**Steps taken**
Hi, I'm trying KFServing local workstation install like shown in GitHub page.

`kind create cluster` runs, but next `./hack/quick_install.sh` throws below error while trying `istioctl verify-install`.

```
(base) kfserving\ $./hack/quick_install.sh 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
100  3015  100  3015    0     0   1465      0  0:00:02  0:00:02 --:--:--  6714
Downloading istio-1.3.6 from https://github.com/istio/istio/releases/download/1.3.6/istio-1.3.6-osx.tar.gz ...  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   612  100   612    0     0    966      0 --:--:-- --:--:-- --:--:--   965
100 36.3M  100 36.3M    0     0  3321k      0  0:00:11  0:00:11 --:--:-- 3822k
Istio 1.3.6 Download Complete!

Istio has been successfully downloaded into the istio-1.3.6 folder on your system.

Next Steps:
See https://istio.io/docs/setup/kubernetes/install/ to add Istio to your Kubernetes cluster.

To configure the istioctl client tool for your workstation,
add the /Users/sunandosamaddar/Local/kfserving/istio-1.3.6/bin directory to your environment path variable with:
         export PATH="$PATH:/Users/sunandosamaddar/Local/kfserving/istio-1.3.6/bin"

Begin the Istio pre-installation verification check by running:
         istioctl verify-install 

Need more information? Visit https://istio.io/docs/setup/kubernetes/install/ 
customresourcedefinition.apiextensions.k8s.io/virtualservices.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/destinationrules.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceentries.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/gateways.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/envoyfilters.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/clusterrbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/policies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/meshpolicies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/rules.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/attributemanifests.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/rbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceroles.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/servicerolebindings.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/adapters.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/instances.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/templates.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/handlers.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/sidecars.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/authorizationpolicies.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/clusterissuers.certmanager.k8s.io created
customresourcedefinition.apiextensions.k8s.io/issuers.certmanager.k8s.io created
customresourcedefinition.apiextensions.k8s.io/certificates.certmanager.k8s.io created
customresourcedefinition.apiextensions.k8s.io/orders.certmanager.k8s.io created
customresourcedefinition.apiextensions.k8s.io/challenges.certmanager.k8s.io created
namespace/istio-system created
poddisruptionbudget.policy/istio-ingressgateway created
poddisruptionbudget.policy/istio-pilot created
serviceaccount/istio-ingressgateway-service-account created
serviceaccount/istio-pilot-service-account created
serviceaccount/istio-multi created
configmap/istio created
clusterrole.rbac.authorization.k8s.io/istio-pilot-istio-system created
clusterrole.rbac.authorization.k8s.io/istio-reader created
clusterrolebinding.rbac.authorization.k8s.io/istio-pilot-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-multi created
role.rbac.authorization.k8s.io/istio-ingressgateway-sds created
rolebinding.rbac.authorization.k8s.io/istio-ingressgateway-sds created
service/istio-ingressgateway created
service/istio-pilot created
deployment.apps/istio-ingressgateway created
deployment.apps/istio-pilot created
horizontalpodautoscaler.autoscaling/istio-ingressgateway created
horizontalpodautoscaler.autoscaling/istio-pilot created
poddisruptionbudget.policy/cluster-local-gateway created
serviceaccount/cluster-local-gateway-service-account created
serviceaccount/istio-multi unchanged
clusterrole.rbac.authorization.k8s.io/istio-reader unchanged
clusterrolebinding.rbac.authorization.k8s.io/istio-multi unchanged
service/cluster-local-gateway created
deployment.apps/cluster-local-gateway created
customresourcedefinition.apiextensions.k8s.io/images.caching.internal.knative.dev created
customresourcedefinition.apiextensions.k8s.io/certificates.networking.internal.knative.dev created
customresourcedefinition.apiextensions.k8s.io/configurations.serving.knative.dev created
customresourcedefinition.apiextensions.k8s.io/ingresses.networking.internal.knative.dev created
customresourcedefinition.apiextensions.k8s.io/metrics.autoscaling.internal.knative.dev created
customresourcedefinition.apiextensions.k8s.io/podautoscalers.autoscaling.internal.knative.dev created
customresourcedefinition.apiextensions.k8s.io/revisions.serving.knative.dev created
customresourcedefinition.apiextensions.k8s.io/routes.serving.knative.dev created
customresourcedefinition.apiextensions.k8s.io/serverlessservices.networking.internal.knative.dev created
customresourcedefinition.apiextensions.k8s.io/services.serving.knative.dev created
customresourcedefinition.apiextensions.k8s.io/images.caching.internal.knative.dev unchanged
namespace/knative-serving created
image.caching.internal.knative.dev/queue-proxy created
configmap/config-autoscaler created
configmap/config-defaults created
configmap/config-deployment created
configmap/config-domain created
configmap/config-gc created
configmap/config-logging created
configmap/config-network created
configmap/config-observability created
configmap/config-tracing created
horizontalpodautoscaler.autoscaling/activator created
deployment.apps/activator created
service/activator-service created
deployment.apps/autoscaler created
service/autoscaler created
deployment.apps/controller created
service/controller created
deployment.apps/webhook created
service/webhook created
clusterrole.rbac.authorization.k8s.io/knative-serving-addressable-resolver created
clusterrole.rbac.authorization.k8s.io/knative-serving-namespaced-admin created
clusterrole.rbac.authorization.k8s.io/knative-serving-namespaced-edit created
clusterrole.rbac.authorization.k8s.io/knative-serving-namespaced-view created
clusterrole.rbac.authorization.k8s.io/knative-serving-admin created
clusterrole.rbac.authorization.k8s.io/knative-serving-core created
clusterrole.rbac.authorization.k8s.io/knative-serving-podspecable-binding created
serviceaccount/controller created
clusterrolebinding.rbac.authorization.k8s.io/knative-serving-controller-admin created
customresourcedefinition.apiextensions.k8s.io/certificates.networking.internal.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/configurations.serving.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/ingresses.networking.internal.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/metrics.autoscaling.internal.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/podautoscalers.autoscaling.internal.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/revisions.serving.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/routes.serving.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/serverlessservices.networking.internal.knative.dev unchanged
customresourcedefinition.apiextensions.k8s.io/services.serving.knative.dev unchanged
validatingwebhookconfiguration.admissionregistration.k8s.io/config.webhook.serving.knative.dev created
mutatingwebhookconfiguration.admissionregistration.k8s.io/webhook.serving.knative.dev created
validatingwebhookconfiguration.admissionregistration.k8s.io/validation.webhook.serving.knative.dev created
secret/webhook-certs created
clusterrole.rbac.authorization.k8s.io/custom-metrics-server-resources created
clusterrolebinding.rbac.authorization.k8s.io/custom-metrics:system:auth-delegator created
clusterrolebinding.rbac.authorization.k8s.io/hpa-controller-custom-metrics created
rolebinding.rbac.authorization.k8s.io/custom-metrics-auth-reader created
deployment.apps/autoscaler-hpa created
service/autoscaler-hpa created
apiservice.apiregistration.k8s.io/v1beta1.custom.metrics.k8s.io created
clusterrole.rbac.authorization.k8s.io/knative-serving-istio created
gateway.networking.istio.io/knative-ingress-gateway created
gateway.networking.istio.io/cluster-local-gateway created
configmap/config-istio created
deployment.apps/networking-istio created
go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go "crd:trivialVersions=true" rbac:roleName=kfserving-manager-role webhook paths=./pkg/apis/... output:crd:dir=config/default/crds/base
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:25:2: cannot find package "github.com/spf13/cobra" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/github.com/spf13/cobra (from $GOROOT)
        /Users/sunandosamaddar/go/src/github.com/spf13/cobra (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:27:2: cannot find package "sigs.k8s.io/controller-tools/pkg/crd" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/crd (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/crd (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:28:2: cannot find package "sigs.k8s.io/controller-tools/pkg/deepcopy" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/deepcopy (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/deepcopy (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:29:2: cannot find package "sigs.k8s.io/controller-tools/pkg/genall" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/genall (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/genall (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:30:2: cannot find package "sigs.k8s.io/controller-tools/pkg/genall/help" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/genall/help (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/genall/help (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:31:2: cannot find package "sigs.k8s.io/controller-tools/pkg/genall/help/pretty" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/genall/help/pretty (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/genall/help/pretty (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:32:2: cannot find package "sigs.k8s.io/controller-tools/pkg/markers" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/markers (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/markers (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:33:2: cannot find package "sigs.k8s.io/controller-tools/pkg/rbac" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/rbac (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/rbac (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:34:2: cannot find package "sigs.k8s.io/controller-tools/pkg/schemapatcher" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/schemapatcher (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/schemapatcher (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:35:2: cannot find package "sigs.k8s.io/controller-tools/pkg/version" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/version (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/version (from $GOPATH)
vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go:36:2: cannot find package "sigs.k8s.io/controller-tools/pkg/webhook" in any of:
        /usr/local/Cellar/go/1.14/libexec/src/sigs.k8s.io/controller-tools/pkg/webhook (from $GOROOT)
        /Users/sunandosamaddar/go/src/sigs.k8s.io/controller-tools/pkg/webhook (from $GOPATH)
make: *** [manifests] Error 1
```

**Environment:**
- Istio Version: 1.3.6
- Kubernetes version: client 1.15.5 server **1.17.0**
- OS: Darwin ... 19.3.0 Darwin Kernel Version 19.3.0: Thu Jan  9 20:58:23 PST 2020; root:xnu-6153.81.5~1/RELEASE_X86_64 x86_64
- Go version: 1.14

--- a/hack/quick_install.sh
+++ b/hack/quick_install.sh
@@ -74,6 +74,7 @@ kubectl apply --filename https://github.com/knative/serving/releases/download/${
 
 cd ..
 # Install KFServing
-export KFSERVING_ENABLE_SELF_SIGNED_CA=true 
-make deploy
+kubectl apply -f install/0.2.2/kfserving.yaml
+
+# Clean up
 rm -rf istio-${ISTIO_VERSION} 

--- a/hack/quick_install.sh
+++ b/hack/quick_install.sh
@@ -2,6 +2,7 @@ set -e
 
 export ISTIO_VERSION=1.3.6
 export KNATIVE_VERSION=v0.12.0
+export KFSERVING_VERSION=0.2.2
 curl -L https://git.io/getLatestIstio | sh -
 cd istio-${ISTIO_VERSION}
 
@@ -74,7 +75,7 @@ kubectl apply --filename https://github.com/knative/serving/releases/download/${
 
 cd ..
 # Install KFServing
-kubectl apply -f install/0.2.2/kfserving.yaml
+kubectl apply -f install/${KFSERVING_VERSION}/kfserving.yaml
 
 # Clean up
 rm -rf istio-${ISTIO_VERSION} 

